docs/SECURITY.md

```diff
@@ -12,6 +12,7 @@ We stick to semantic versioning, so unless the major version changes, updating C
 | - | - | - | - |
 | initContainers not validated | <span>&#8804;</span> 1.3.0 | 1.3.1 | Prior to version 1.3.1 Connaisseur did not validate initContainers which allowed deploying unverified images to the cluster. |
 | Ephemeral containers not validated | <span>&#8804;</span> 3.1.1 | 3.2.0 | Prior to version 3.2.0 Connaisseur did not validate ephemeral containers (introduced in k8s 1.25) which allowed deploying unverified images to the cluster. |
+| Regex Denial of Service for Notary delegations | <span>&#8804;</span> 3.3.0 | 3.3.1 | Prior to version 3.3.1 Connaisseur did input validation on the names of delegations in an unsafe manner: A network adversary or evil Notary server could have provided Connaisseur with an invalid delegation name that would lead to catastrophic backtracking during a regex matching. Only users of type `notaryv1` validators are affected as Connaisseur will only perform this kind of input validation in the context of a Notary validation. The default configuration is vulnerable as it contains a `notaryv1` validator with the root keys of both Connaisseur and the library of official Docker images. |
```

Comment:
As discussed, network attackers are in default configuration very unlikely because hosts are specified and are using TLS, so the last sentence is probably formulated too strong